### PR TITLE
gluon-status-page: add mesh protocol to status-page

### PR DESCRIPTION
**Pull request by AiyionPrime**
This is a draft to show the deployed mesh protocol as well as its version on the routers statuspage.

Its addresses another open point on the agenda of #1415.

An example router can be found at http://lilienstra-e.n.ffh.zone/, or in case our dns misbehaves http://[2001:678:978:114:b64b:d6ff:fe20:2778]/cgi-bin/status .

### CODE REVIEW

**Comment by AiyionPrime** [comment]
This is the current solution as a oneliner:

img

**Comment by AiyionPrime** [comment]
One alternative in firmware:

img

**Comment by AiyionPrime** [comment]
And another draft below `Firmware`:

img

File: package/gluon-status-page/files/lib/gluon/status-page/view/status-page.html [hunk 1]
```diff
@@ -136,6 +136,15 @@ <h2><%:Overview%></h2>
 								string.format(' (%s)', nodeinfo.software.autoupdater.branch)
 						%></dd>
 					<%- end %>
+					<% if nodeinfo.software.babeld or nodeinfo.software['batman-adv'] then -%>
+						<dt><%:Mesh protocol%></dt>
+						<% if nodeinfo.software.babeld then -%>
+						<dd>babel <%| nodeinfo.software.babeld.version %></dd>
```
Comment (mweinelt) [inline package/gluon-status-page/files/lib/gluon/status-page/view/status-page.html:142]:
```suggestion
						<dd>babeld <%| nodeinfo.software.babeld.version %></dd>
```

Comment (AiyionPrime) [reply package/gluon-status-page/files/lib/gluon/status-page/view/status-page.html]: resolved in d2a5b3f9c187ecaeb232bf9b82e574247fc9e7fa

**Comment by mweinelt** [comment]
Do you know "git commit --fixup"? Not sure where "squashme:" actually comes from.

**Comment by AiyionPrime** [comment]
I didn't. Thanks.

File: package/gluon-status-page/files/lib/gluon/status-page/view/status-page.html (hunk 1 above)
Comment (AiyionPrime) [reply package/gluon-status-page/files/lib/gluon/status-page/view/status-page.html]: fixup in 77a6b9a947759ed6dad826609535f7667ef47265

**Comment by lemoer** [comment]
LGTM.

**Review by lemoer — APPROVED** [review]

File: package/gluon-status-page/files/lib/gluon/status-page/view/status-page.html [hunk 2]
```diff
@@ -136,6 +136,15 @@ <h2><%:Overview%></h2>
 								string.format(' (%s)', nodeinfo.software.autoupdater.branch)
 						%></dd>
 					<%- end %>
+					<% if nodeinfo.software.babeld or nodeinfo.software['batman-adv'] then -%>
```
Comment (mweinelt) [inline package/gluon-status-page/files/lib/gluon/status-page/view/status-page.html:139]: What is there difference between these two lookups?

**Review by mweinelt — APPROVED** [review]

**Pull request merged**